src/cms/templates/regions/region_list.html

```diff
@@ -22,9 +22,9 @@ <h2>
 </div>
 
 <div class="table-listing">
-    <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+    <table class="w-full mt-4 rounded border border-solid border-gray-400 shadow bg-white">
```

Comment:
I don't like this dark border very much. I would rather suggest to make the border of the POI list more light instead of all other list dark.